api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClientImpl.java

```diff
@@ -218,20 +240,44 @@ public void createRuntime(
         WORKSPACE_CDR_ENV_KEY,
         cdrVersion.getBigqueryProject() + "." + cdrVersion.getBigqueryDataset());
 
+    String datasetsBucket = cdrVersion.getAccessTier().getDatasetsBucket();
+    String bucketInfix = cdrVersion.getStorageBasePath();
+    if (!Strings.isNullOrEmpty(datasetsBucket) && !Strings.isNullOrEmpty(bucketInfix)) {
+      String basePath = joinStoragePaths(datasetsBucket, bucketInfix);
+      customEnvironmentVariables.putAll(
+          ImmutableMap.of(
```

Comment:
nit: For Map that has more than 2 entry, `MapBuilder.put().put().build` pattern is more clear:
https://cs.github.com/all-of-us/gatk/blob/ac7bec0beac7b3ea042ca3177834e466d2dfc19a/src/test/java/org/broadinstitute/hellbender/engine/spark/SparkContextFactoryUnitTest.java?q=%22ImmutableMap.Builder%22#L21